frontend/src/use-cases/category/elements/OpeningHours.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "styled-components";
+
+import { Text } from "../../../common-ui/text";
+import { Spacing } from "../../../common-ui/layout";
+
+/**
+ * openingHours is an array, every element in the array should have the following:
+ * - startDay (String)
+ * - endDay (String)
+ * - openingHours (String)
+ * - closingHours (String)
+ */
+
+export const OpeningHours = ({ openingHours }) => (
+  <div>
+    <Text>Öppetider:</Text>
+    <Spacing />
+    <Row>
+      <Column align="right">
+        {openingHours.map(item => (
+          <Cell key={item.startDay}>
+            {item.startDay === item.endDay ? (
+              <Text key={item.startDay}>{item.startDay + ":"}</Text>
+            ) : (
+              <Text key={item.startDay}>
+                {item.startDay + " - " + item.endDay + ":"}
+              </Text>
+            )}
+            <Spacing />
+          </Cell>
+        ))}
+      </Column>
+      <Spacing />
+      <Column align="left">
+        {openingHours.map(item => (
+          <Cell key={item.startDay + item.openingHours}>
+            {item.closingHours == null ? (
+              <Text key={item.startDay + item.openingHours}>
+                {item.openingHours}
```

Comment:
Same as previous comment in file!

Unify these 2 cases in one formatter! Move formatting logic out of rendering!